qutebrowser/browser/commands.py

```diff
@@ -1748,3 +1756,19 @@ def fullscreen(self, leave=False, enter=False):
 
         log.misc.debug('state before fullscreen: {}'.format(
             debug.qflags_key(Qt, window.state_before_fullscreen)))
+
+    def _possibly_open_in_existing_tab(self, url):
```

Comment:
This is used in app.py, so don't mark it as private.

Reply:
Good catch